ocaml/quicktest/qt_filter.ml

```diff
@@ -187,6 +187,11 @@ module SR = struct
   let smapiv3 =
     sr_filter (fun i -> not (is_smapiv1 i))
 
+  let has_type types sr_info =
+    List.mem (Client.Client.SR.get_type ~rpc:!A.rpc ~session_id:!session_id ~self:sr_info.Qt.sr) types
+
+  let thin_pro = sr_filter (has_type ["gfs2"; "nfs"; "smb"; "ext"; "file"])
```

Comment:
Would be nice if the SR themselves exposed some capability that helped us tell the difference.

Reply:
I don't think they do, sadly, I've checked the SR and SM fields.  Is there a way to figure this out @stefanopanella ?

Reply:
Since this is just test code it is probably fine to hardcode, just wanted to check there isn't another way.

Reply:
Stefano said this is the right approach. It's best to just filter based on SR type.